core/src/main/web/app/beaker.js

```diff
@@ -272,18 +272,21 @@
       }
     });
 
-    beaker.run(function($location, $route, $document, bkUtils, bkCoreManager, bkHelper) {
+    beaker.run(function($rootScope, $location, $route, $document, bkUtils, bkCoreManager, bkHelper) {
       var user;
       var lastAction = new Date();
       var beakerRootOp = {
         gotoControlPanel: function() {
-          return $location.path("/control").search({});
+          var ret = $location.path("/control").search({});
+          if (bkUtils.isElectron) {
+            $rootScope.$apply();
```

Comment:
No clue, but without the return value the redirection fails. I don't really know much about angular